specs/driver_spec.rb

```diff
@@ -77,4 +77,69 @@
       driver.average_rating.must_equal 0
     end
   end
+  describe "1.2 Aggregate Statistics" do
+    before do #use pry to get this info as hard coded data
+      dispatcher = RideShare::TripDispatcher.new
+      trip = dispatcher.trips.first
+      @driver = trip.driver
+      @duration = trip.duration
+    end
+
+    describe "Driver#total_revenue" do
+      it "returns total revenue" do
+        # use hard coded data to created expected output, not magic numbers
```

Comment:
For all the wave 1 methods (on both `Driver` and `Passenger`), I would also be interested in what happens if the driver or passenger has no trips.